Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:alpine
+
+RUN apk --update add --virtual build_deps \
+    build-base \
+    libc-dev \
+    linux-headers \
+    cmake \
+    && apk --no-cache add icu-dev libressl-dev \
+    && gem install github-linguist \
+    && apk del build_deps build-base libc-dev linux-headers
```

Comment:
What about removing cmake, icu-dev, and libressl-dev?

Reply:
Removed `cmake`, but removing `icu-dev` or `libressl-dev` causes errors when trying to run  `linguist`
<details>
Traceback (most recent call last):
	16: from /usr/local/bundle/bin/github-linguist:23:in `<main>'
	15: from /usr/local/bundle/bin/github-linguist:23:in `load'
	14: from /usr/local/bundle/gems/github-linguist-7.6.1/bin/github-linguist:5:in `<top (required)>'
	13: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	12: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	11: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist.rb:6:in `<top (required)>'
	10: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 9: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 8: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist/repository.rb:1:in `<top (required)>'
	 7: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 6: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 5: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist/lazy_blob.rb:3:in `<top (required)>'
	 4: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 3: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 2: from /usr/local/bundle/gems/rugged-0.28.3.1/lib/rugged.rb:8:in `<top (required)>'
	 1: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- rugged/2.6/rugged (LoadError)
	17: from /usr/local/bundle/bin/github-linguist:23:in `<main>'
	16: from /usr/local/bundle/bin/github-linguist:23:in `load'
	15: from /usr/local/bundle/gems/github-linguist-7.6.1/bin/github-linguist:5:in `<top (required)>'
	14: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	13: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	12: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist.rb:6:in `<top (required)>'
	11: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	10: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 9: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist/repository.rb:1:in `<top (required)>'
	 8: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 7: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 6: from /usr/local/bundle/gems/github-linguist-7.6.1/lib/linguist/lazy_blob.rb:3:in `<top (required)>'
	 5: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 4: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 3: from /usr/local/bundle/gems/rugged-0.28.3.1/lib/rugged.rb:6:in `<top (required)>'
	 2: from /usr/local/bundle/gems/rugged-0.28.3.1/lib/rugged.rb:10:in `rescue in <top (required)>'
	 1: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': Error loading shared library libssl.so.45: No such file or directory (needed by /usr/local/bundle/gems/rugged-0.28.3.1/lib/rugged/rugged.so) - /usr/local/bundle/gems/rugged-0.28.3.1/lib/rugged/rugged.so (LoadError)
</details>

Reply:
Ah. Weird, I didn't get the same errors on my setup.
Let's leave them then!